ui/src/pageLayout/containers/Nav.test.tsx

```diff
@@ -2,29 +2,38 @@
 import React from 'react'
 import {render} from 'react-testing-library'
 import {IconFont} from 'src/clockface'
+import _ from 'lodash'
 
 // Components
 import NavMenu from 'src/pageLayout/components/NavMenu'
 
+// Constants
+const DASHBOARDS_NAV_ITEM = 'Dashboards'
+const TASKS_NAV_ITEM = 'Tasks'
+
 function Nav(props) {
   const {pathname} = props
+
+  const activateNavItem = (keywords: string[]): boolean => {
+    const parentPath = _.get(pathname.split('/'), '3', '')
```

Comment:
what is this `3`? could you name this variable so its obvious what is happening here?

Reply:
I actually have no idea what the 3 does it was there before, I just moved the code to a different place. I'll ask @121watts about it since I think he wrote it

Reply:
so it skips the first two items in the URL which are currently always org and org ID